com.github.akiraux.akira.json

```diff
@@ -0,0 +1,67 @@
+{
+   "app-id": "com.github.akiraux.akira",
+   "runtime": "org.gnome.Platform",
+   "runtime-version": "3.30",
+   "sdk": "org.gnome.Sdk",
+   "base":"io.elementary.BaseApp",
+   "base-version": "juno",
+   "command": "com.github.akiraux.akira",
+   "cleanup": [
+      "/include",
+      "/lib/pkgconfig",
+      "/lib/debug",
+      "/share/vala",
+      "/man",
+      "*.a",
+      "*.la"
+   ],
+   "finish-args": [
+      "--share=ipc",
+      "--socket=x11",
+      "--socket=wayland",
+      "--share=network",
+      "--filesystem=host",
```

Comment:
The app allows to open just one file at once or multiple files, but we don't open folders or anything like that. We just need to use the NativeFileChooser to use portals and remove this permission

Reply:
I'm thinking about config files for akira or similar

Reply:
Config files are stored under `.config` and apps have the rights for that.

Reply:
ok